src/python_bindings/gfd/bind_gfd_mining.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#include <pybind11/pybind11.h>
+
+namespace python_bindings {
+void BindGfdMining(pybind11::module_& main_module);
```

Comment:
Maybe we can simplify name `BindGfd`? For consistency with other primitives.
(don't forget to change file name)

Reply:
There is already the GFD verification primitive, thus I doubt shortening the name would make a lot of sense